freemocap/fmc_good_frame_finder.py

```diff
@@ -0,0 +1,103 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+
+
+def find_velocity_values_within_limit(skeleton_velocity_data, velocity_limit):
+    """
+    This function takes in a skeleton velocity data array and a limit and returns the indices of the values that are within the limit
+    """
+    indices = []
+    for i in range(len(skeleton_velocity_data)):
```

Comment:
At some point, remind me to show you how to do this with logical indexing

this method totally works, but it's a tad bit slower than it needs to be (not by much, but probably noticable with bigger larger number)